Examples/IconExplorer/android/app/src/main/java/com/iconexplorer/MainApplication.java

```diff
@@ -24,6 +26,8 @@ public boolean getUseDeveloperSupport() {
     protected List<ReactPackage> getPackages() {
       return Arrays.<ReactPackage>asList(
           new MainReactPackage(),
+            new RNScreensPackage(),
+            new RNGestureHandlerPackage(),
```

Comment:
Is not all [steps](https://kmagiera.github.io/react-native-gesture-handler/docs/getting-started.html#android) necessary on android?

Reply:
Fixed in https://github.com/oblador/react-native-vector-icons/pull/1001/commits/38a2c9e0436f00e4ac66672f7e5be0593e9350f9